cellengine/resources/population.py

```diff
@@ -1,44 +1,46 @@
 from __future__ import annotations
-from dataclasses import field
-from typing import Optional
+from typing import Dict, Optional
+from attr import define, field
 
-from dataclasses_json.cfg import config
-from cellengine.utils.dataclass_mixin import DataClassMixin, ReadOnly
-
-from dataclasses import dataclass
 import cellengine as ce
+from cellengine.utils import converter
+from cellengine.utils.readonly import readonly
 
 
-@dataclass
-class Population(DataClassMixin):
+@define
+class Population:
+    _id: str = field(on_setattr=readonly)
+    experiment_id: str = field(on_setattr=readonly)
     name: str
     gates: str
+    unique_name: Optional[str] = field(default=None, on_setattr=readonly)
     parent_id: Optional[str] = None
     terminal_gate_gid: Optional[str] = None
-    _id: str = field(
-        metadata=config(field_name="_id"), default=ReadOnly()
-    )  # type: ignore
-    experiment_id: str = field(default=ReadOnly())  # type: ignore
-    unique_name: str = field(default=ReadOnly())  # type: ignore
 
     def __repr__(self):
         return f"Population(_id='{self._id}', name='{self.name}')"
 
-    @classmethod
-    def get(cls, experiment_id: str, _id: str = None, name: str = None):
-        kwargs = {"name": name} if name else {"_id": _id}
-        return ce.APIClient().get_population(experiment_id, **kwargs)
+    @property
+    def client(self):
+        return ce.APIClient()
```

Comment:
(ditto other PR)

Reply:
These were in my brain as "already deleted", but it must have been lost in the mess of the other PR. Anyway, :+1: